msg/CartesianTrajectory.msg

```diff
@@ -0,0 +1,6 @@
+Header header
```

Comment:
GenericTrajectory.msg already has a Header so I don't think we need it duplicated here

Reply:
I think `CartesianTrajectory` should be self-contained. At the same time, `trajectory_msgs/JointTrajectory` also has a header so there's an option of removing the header from `GenericTrajectory`. What do you think of that?